src/bin/rav1e.rs

```diff
@@ -162,6 +162,15 @@ fn main() {
   cli.enc.bit_depth = video_info.bit_depth;
   cli.enc.chroma_sampling = video_info.chroma_sampling;
   cli.enc.chroma_sample_position = video_info.chroma_sample_position;
+
+  // If no pixel range is specified via CLI, assume limited,
```

Comment:
Do we want to be able to explicitly set the "Unspecified" value from the cli? (seems useless to me but we do lose that ability now)

Reply:
What is the behaviour in the encoder if it is unspecified? I also thought it was not worth adding an extra parameter to `CliOptions`, but we can do that if preferred.

Reply:
I think unspecified is actually a valid field option in the AV1 header.

Reply:
(then it's up to the player to decide. tbh this behavior seems useless to me)

Reply:
> I think unspecified is actually a valid field option in the AV1 header.

Indeed.

While it doesn't seem very useful, if someone did want to specify the range as "unspecified", it would be surprising for it to be changed to "limited". So for the sake of avoiding oddities for end users I have pushed a commit to honor a user's choice to leave the range as unspecified. Either way is fine by me though, so let me know if I should roll it back.

Reply:
Seems best this way.